tests/functional/r/regression/regression_2913.py

```diff
@@ -14,6 +14,6 @@ class BaseCorrect:  # pylint: disable=too-few-public-methods
     """My base class."""
 
 
-class BaseInner:  # [too-few-public-methods]
+class BaseInner:
```

Comment:
This was actually inconsistent with how we describe this should work, see:
https://github.com/PyCQA/pylint/blob/b31dd5623966069386b9df3ccc08e2834aca621d/pylint/utils/file_state.py#L70-L85

According to this, a disable on L18 should affect L17. Apparently it did not, and this got fixed with this change.

The original issue actually discusses this as well:
https://github.com/PyCQA/pylint/issues/2913

The reporter wonders why this isn't behaving like our internal comment is telling it should. I think this is thus a good change (and perhaps worthy of a changelog).